src/content/pages/capi.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: C API Summit
+subtitle: A summit to bring together users (and interested people) of the C API that Python provide, to discuss the state of this ecosystem, existing challenges, and ongoing work.
+---
+
+# C API Summit
+
+* **When**: Monday, July 8th.
+* **Where**: Prague Congress Centre (PCC), Room Club D
+* **Who can join**: Anyone with a valid in-person EuroPython 2024 ticket - Conference, Tutorial or Combined (see [ticket types](https://ep2024.europython.eu/tickets#ticket-types) for details).
+
+The C API has many different stakeholders, including but not limited to
+CPython core developers, maintainers of Python extensions, code generators
+(Cython, pybindy11, etc) and alternative API design (HPy)
+
+The main goal of this summit is to make sure that people from all these
+different projects can meet and discuss about the state of the C API, existing
+challenges and ongoing work.
+
+## Agenda
+
+* 9:00: Meet and greet - nconference-y post-it based organisation.
+* 9:30: Presentations (30mins each)
+* 11:00: Coffee
+* 11:20: Presentations
+* 13:00: Lunch at the PCC (included)
+* 14:00: Unconference-y activities (discussions, hacks, ad hoc tutorials etc…)
+* 16:00: Round-up / plenary session for feedback and organising next steps.
+* 18:30: Dinner
+
+## Registration
+
+You need to have a valid EuroPython in-person ticket ([Conference, Tutorial or Combined](https://ep2024.europython.eu/tickets#ticket-types)) to participate. Purchase a EuroPython ticket [here](https://tickets.europython.eu/), if you haven't already.
+
+The event is limited to 40 participants. If there is a topic you would like to present, please indicate it in the form and _fill it in early_. Time slots are of 30min at most (10 min of presentation + 20 min for questions/discussion).
+
+To be part of the C API summit, [register your interest now!](https://)
```

Comment:
```suggestion
To be part of the C API summit, [register your interest now!](https://docs.google.com/forms/d/e/1FAIpQLSc2tLqibsxg-GhTOYkdcMtSopRwojct8-lqhxAE-F9a7Xghww/viewform)
```

Reply:
I still see on the form:
> This summit will take place on July XXX, 2024 at EuroPython 2023 in Prague, Czech Republic.
the XXX needs to replaced

Reply:
done!